File: 1.baekjoon/B (31 ~ 60)/S_4344_38%.py
import sys
MIS = lambda : map(int,sys.stdin.readline().split())

def fake(cnt, pp):
    avg = sum(pp)/cnt
    up = sum([1 for i in pp if i > avg])
    num = up/cnt*100
    msg = '%.3f' %num + '%'
    print(msg)
    
if __name__ == "__main__":
    T = int(input())
    for _ in range(T):
        list1 = list(MIS())
        fake(list1[0],list1[1:])

File: 1.baekjoon/B (31 ~ 60)/S_4344_38%.cpp
#include <iostream>
using namespace std;

double avg()
{
  int num, cnt=0;
  double avg = 0;
  cin>>num;
  int arr[1000];
  for(int i=0; i<num; ++i)
  {
    cin>>arr[i];
    avg+=arr[i];
  }
  avg = avg/num;
  for(int i=0; i<num; ++i)
    if(arr[i] > avg)
      cnt++;
  return (double)cnt/num*100;
}

int main(void)
{
  int n;
  cin>>n;
  cout.precision(3);
  for(int i=0; i<n; ++i)
  {
    cout<<fixed<<avg()<<"%"<<endl;
  }
}